polkadot/.rpm/polkadot.spec

```
@@ -37,6 +37,7 @@ getent passwd polkadot >/dev/null || \
 if [ ! -e "$config_file" ]; then
     echo 'POLKADOT_CLI_ARGS=""' > /etc/default/polkadot
 fi
+polkadot key generate-node-key --default-base-path
```

```suggestion
if [ $1 -eq 1 ]; then
    polkadot key generate-node-key --default-base-path
fi
```

Only do that when installing, not when upgrading the package